src/components/EditableStudentGroupName.tsx

```diff
@@ -13,6 +13,10 @@ const EditableStudentGroupName: React.FC<EditableStudentGroupNameProps> = ({
 }) => {
   const editStudentGroupNameHandler = (value: string) => {
     console.log(value)
+    if (value.length > 22) {
```

Comment:
is this just a custom validation?  better to validate through react-hook-form, no?

Reply:
Yeah, if I end up doing this.  Do you think it's an okay idea to limit the characters in a group name for display purposes?

Reply:
Okay, after a long struggle I got the onSubmit working for the Editable chakra input but now the validation is making the input behave weird (keypresses are not registering consistently as error messages are displayed).  I'm starting to wonder if using these two libraries together is too big of a pain in this particular case.  Since the Editable component is just a single input, would it be okay to just do my own validation in this case?  I would still use chakra for the error messages so the styling would be consistent.  Then I would use react-hook-form for any real forms, like with normal inputs and a submit button.  What do you think?